coreTable/CoreTable.ts

```diff
@@ -283,12 +283,7 @@ export class CoreTable<
             )
         }
 
-        if (
-            inputType === InputType.ColumnStore ||
```

Comment:
I got rid of this check in order to force parsing to happen on a JSON column store. Otherwise the `date` column was ending up unparsed and caused a crash.

Reply:
Could have obvious negative effects: a `columnStore` being reparsed unnecessarily in some case

Reply:
Makes sense to me; I guess every `columnStore` that doesn't need to be parsed should have a `parent` set anyway.

Reply:
The only concerning thing to me is that the test failure (in the LineChart test) we get is using this exact use case - a `ColumnStore` object that is passed as input to a table, and apparently some values go missing/haywire in the processing done here?
Might be worth a good look to see what's going wrong there.

Reply:
Yeah. I will also set up a `if (inputType === InputType.ColumnStore) console.log("was going to skip it but thanks to you now users suffer")` to test